go2rtc_client/ws/client.py

```diff
@@ -128,16 +134,8 @@ async def _receive_messages(self) -> None:
                         self._process_text_message(msg.data)
                     case _:
                         _LOGGER.warning("Received unknown message: %s", msg)
-        except Exception:
-            _LOGGER.exception("Unexpected error while receiving message")
-            raise
-        finally:
-            _LOGGER.debug(
-                "Websocket client connection from %s closed", self._server_url
-            )
-
-            if self.connected:
-                await self.close()
+            except Exception:  # pylint: disable=broad-except
```

Comment:
In theory this should never happen

https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/client_ws.py#L355

Reply:
I guess its possible that `close()` gets called and then its already closing and https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/_websocket/writer.py#L125 is hit

Reply:
nope we swallow that as well https://github.com/aio-libs/aiohttp/blob/5f61c7bff5bd0560a169b611f5c4b7e4b1ee3360/aiohttp/client_ws.py#L278

Reply:
So it _should_ never raise here